changelog.md

```diff
@@ -9,6 +9,8 @@
 
 ## Bug fixes
 
+* Fix training Node regession pipelines created via `gds.alpha.pipeline.nodeRegression.create`.
```

Comment:
I dont understand what bug this is fixing. Looks more like the whole feature was not exposed ?

Reply:
It is a bug, as the feature was documented and therefore exposed